test/skills/core.py

```diff
@@ -0,0 +1,99 @@
+import unittest
+from os.path import join, dirname
+
+from mycroft.skills.core import load_regex_from_file, load_regex
+from mycroft.util.log import getLogger
+
+__author__ = 'eward'
+logger = getLogger(__name__)
+
+
+class MockEmitter(object):
+    def __init__(self):
+        self.types = []
+        self.results = []
+
+    def emit(self, message):
+        self.types.append(message.message_type)
+        self.results.append(message.metadata['regex'])
+
+    def get_types(self):
+        return self.types
+
+    def get_results(self):
+        return self.results
+
+    def reset(self):
+        self.types = []
+        self.results = []
+
+
+class MycroftSkillTest(unittest.TestCase):
```

Comment:
There is only one missing test it'd be great to have too. That's for the case when **no valid regex** is provided in a .rx file which means the `re.compile()` would fail. Please, add this one.